src/graphql/error-map.ts

```diff
@@ -30,10 +30,21 @@ const assertUnreachable = (x: never): never => {
   throw new Error(`This should never compile with ${x}`)
 }
 
-export const mapError = (error: ApplicationError): CustomApolloError => {
+export const mapError = (err: ApplicationError | unknown): CustomApolloError => {
```

Comment:
This should never be `unknown` that would be a significant bug!

Reply:
@vindard and i synced and created a new `error-parser` called `parseUnknownDomainErrorFromUnknown` which returns a `DomainError`.  So now this works in mapError with a new fallback type called `UnknownDomainError`